charts/images.yaml

```diff
@@ -25,5 +25,5 @@ images:
   tag: 1.7.1
 - name: calico-cpva
   sourceRepository: github.com/kubernetes-incubator/cluster-proportional-vertical-autoscaler
-  repository: k8s.gcr.io/cpvpa-amd64
-  tag: v0.8.1
\ No newline at end of file
+  repository: eu.gcr.io/k8s-artifacts-prod/cpa/cpvpa-amd64
```

Comment:
FYI - https://groups.google.com/forum/#!msg/kubernetes-dev/MkXnkTSJ_vs/IeI3LgmDCAAJ

--> Is it valid to use `k8s-artifacts-prod` GCR already?

Reply:
See https://github.com/kubernetes-incubator/cluster-proportional-vertical-autoscaler/issues/33#issuecomment-612258918.

Reply:
is this a temporary domain for the registry, i.e., will the image always be accessible at that place or will we have to change that later? The thread is not clear about this point.

Reply:
@MrHohn, @listx, we can proceed with the `eu.gcr.io/k8s-artifacts-prod/cpa/cpvpa-amd64:v0.8.2` image, right?
When the flip for the vanity domain happens, we can switch back to `k8s.gcr.io/cpa/cpvpa-arm64:v0.8.2`, right?

Reply:
@ialidzhikov That's correct.

@zanetworker The `k8s.gcr.io` domain will probably be around a very, very long time. We've gone to very great lengths to try to keep it alive, and will continue to do so for the foreseeable future.

Reply:
thanks for the swift response :)

Reply:
I guess this here is also the same discussion https://github.com/kubernetes/kubernetes/pull/90093#discussion_r407580184

Reply:
Yes [kubernetes/kubernetes#90093 (comment)](https://github.com/kubernetes/kubernetes/pull/90093#discussion_r407580184) seems to be the same; I commented there also.

> thanks for the swift response :)

NP